pynumdiff/optimize/_optimize.py

```diff
@@ -4,96 +4,101 @@
 from itertools import product
 from functools import partial
 from warnings import filterwarnings, warn
-from multiprocessing import Pool
+from multiprocessing import Pool, Manager
+from hashlib import sha1
 from tqdm import tqdm
 
 from ..utils import evaluate
 from ..finite_difference import finitediff, first_order, second_order, fourth_order
 from ..smooth_finite_difference import mediandiff, meandiff, gaussiandiff, friedrichsdiff, butterdiff
-from ..linear_model import lineardiff
 from ..polynomial_fit import polydiff, savgoldiff, splinediff
 from ..basis_fit import spectraldiff, rbfdiff
 from ..total_variation_regularization import tvrdiff, velocity, acceleration, jerk, iterative_velocity, smooth_acceleration, jerk_sliding
-from ..kalman_smooth import rtsdiff, constant_velocity, constant_acceleration, constant_jerk
-
+from ..kalman_smooth import rtsdiff, constant_velocity, constant_acceleration, constant_jerk, robustdiff
+from ..linear_model import lineardiff
 
 # Map from method -> (search_space, bounds_low_hi)
 method_params_and_bounds = {
-    spectraldiff: ({'even_extension': {True, False}, # give categorical params in a set
```

Comment:
I changed the order of a bunch of listings around this PR, to better match the taxonomy paper and readme.